boards/msbiot/include/periph_conf.h

```diff
@@ -27,6 +27,24 @@
 extern "C" {
 #endif
 
+/**
+ * @name    DMA streams configuration
+ * @{
+ */
+#ifdef MODULE_PERIPH_DMA
+static const dma_conf_t dma_config[] = {
+    { .stream = 11 },   /* DMA2 Stream 3 - SPI1_TX */
```

Comment:
How do you determine the value set to stream ? I can't find anything related to 11 or 10 in the reference manual. Shouldn't it be 3 here and 2 below ?

Reply:
Oh, I think I have it now: since it's DMA2, the 8 streams of DMA1 are added. So makes sense.

Reply:
Yes, it is not completely intuitive, we should probably add a `DMA_STREAM(dma_dev, stream)` macro at some point for clarity. It is described [here](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32_common/include/periph_cpu_common.h#L273-L297) and [here](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32_common/periph/dma.c#L99-L107)